partner_duns/models/res_partner.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Camptocamp SA (https://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    duns = fields.Char(
+        help="Data Universal Numbering System",
```

Comment:
```suggestion
    duns = fields.Char(
        string="DUNS",
        help="Data Universal Numbering System",
```

Reply:
Totally agree, thanks for the suggestion!
I've just changed it